components/k8s-dashboard-proxy/README.md

```diff
@@ -0,0 +1,32 @@
+# Kubernetes Dashboard proxy
+
+## Overview
+
+The Kubernetes Dashboard proxy is a transparent proxy for the Kubernetes Dashboard allowing the Kyma clients to control the access to the Kubernetes Dashboard. Even though by default you don't need any verification to access the the Dashboard, the proxy uses Dex identity provider to ensure the Dashboard users are authenticated and authorized.
```

Comment:
`The Kubernetes Dashboard proxy is a transparent proxy allowing the Kyma clients to control the access to the Kubernetes Dashboard. Even though by default you don't need any verification to access the the Dashboard, the proxy uses Dex identity provider to ensure the Dashboard users are authenticated and authorized.`

Small fix to remove duplicate "the" and simplify the first sentence.

Reply:
OK!
It will be like that
> Kubernetes Dashboard proxy is a transparent proxy allowing the Kyma clients to control the access to the Kubernetes Dashboard.